CMakeLists.txt

```diff
@@ -143,12 +143,21 @@ install(FILES "${COMPONENT_HASH_PATH}"
 #
 # share/revng
 #
-add_custom_target(copy_share ALL COMMAND cp -Tar "${CMAKE_SOURCE_DIR}/share/"
-                                         "${CMAKE_BINARY_DIR}/share/")
+add_custom_target(
+  copy_share ALL
+  COMMAND mkdir -p "${CMAKE_BINARY_DIR}/share/"
+  COMMAND cp -Tar "${CMAKE_SOURCE_DIR}/share/doc"
+          "${CMAKE_BINARY_DIR}/share/doc"
+  COMMAND cp -Tar "${CMAKE_SOURCE_DIR}/share/revng"
+          "${CMAKE_BINARY_DIR}/share/revng")
```

Comment:
Why? The whole `share/` directory is designed to be copied as is.
Please reorganize things so that things stay as they are, or come and we can discuss.